packages/flutter/lib/src/material/dialog.dart

```diff
@@ -60,6 +61,14 @@ class Dialog extends StatelessWidget {
   /// Defaults to [Curves.fastOutSlowIn].
   final Curve insetAnimationCurve;
 
+  /// The semantic label of the dialog used by accessibility frameworks
```

Comment:
Maybe be a little more descriptive right here how the value is used? E.g. to announce screen transitions when the dialog is brought on screen?